package.json

```diff
@@ -77,15 +78,26 @@
     "postcss-preset-env": "^9.5.14",
     "rehype-highlight": "^7.0.0",
     "rehype-stringify": "^10.0.1",
-    "remark": "^15.0.1",
+    "remark-cli": "10.0.1",
+    "remark-copy-linked-files": "^1.5.0",
+    "remark-frontmatter": "^4.0.1",
+    "remark-gfm": "^3.0.1",
+    "remark-lint-no-dead-urls": "^1.1.0",
+    "remark-mdx": "^2.1.1",
+    "remark-parse": "^10.0.1",
+    "remark-preset-lint-markdown-style-guide": "^5.1.2",
+    "remark-rehype": "^10.1.0",
+    "remark-validate-links": "^11.0.2",
     "to-vfile": "^8.0.0",
+    "tsc-esm-fix": "^3.1.0",
     "tsm": "^2.3.0",
     "typescript": "~5.5.2",
     "unified": "^11.0.5",
     "unified-lint-rule": "^3.0.0",
     "unist": "^0.0.1",
-    "unist-util-find": "^3.0.0",
-    "unist-util-visit-parents": "^6.0.1",
+    "unist-util-find": "^1.0.2",
+    "unist-util-visit": "4.1.0",
+    "unist-util-visit-parents": "^5.1.0",
```

Comment:
Seems like some dependencies got downgraded

Reply:
I was trying to replicate our `gravitational/docs` configuration to minimize new linting errors. I had to fix new linting errors anyway, so I've reverted these downgrades